website/docs/components/accordion/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Accordion
+description: An accordion is a vertically stacked list of container-like toggles that reveal or hide associated sections of content.
+caption: A list of toggles that reveal or hide associated sections of content.
+status: released
```

Comment:
```suggestion
```

I just prioritized a ticket for cleaning this up more holistically in this sprint, but going forward we don't need this information (it's not used).